### PR TITLE
Ms.final constants

### PR DESCRIPTION
**Pull request by mariano54**
Merge for mainnet release

### CODE REVIEW

File: src/cmds/init.py [hunk 1]
```diff
@@ -312,6 +312,12 @@ def chia_init(root_path: Path):
         )
 
     print(f"Chia directory {root_path}")
+    if root_path.is_dir() and Path(root_path / "config" / "config.yaml").exists():
+        # This is reached if CHIA_ROOT is set, or if user has run chia init twice
+        # before a new update.
+        check_keys(root_path)
+        print(f"{root_path} already exists, no migration action taken")
+        return -1
```
Comment (hoffmang9) [inline src/cmds/init.py:320]: This needs the same change we made right before RC9

**Review by hoffmang9 — COMMENTED** [review]
farming rewards addresses encode differently with an xch prefix, right? Doesn't that mean we need to set them fresh?

**Comment by mariano54** [comment]
The bech32m encoding (address) is slightly different, but the puzzle hash is the same, regardless of prefix

File: src/cmds/init.py (hunk 1 above)
Comment (mariano54) [reply src/cmds/init.py:320]: :+1:

Comment (mariano54) [reply src/cmds/init.py:320]: Wait, this reverts it to the old behaviour, which is correct I think.

**Comment by mariano54** [comment]
Replaced by https://github.com/Chia-Network/chia-blockchain/pull/1351

**Pull request closed without merging**